moveit_core/collision_detection/include/moveit/collision_detection/world.h

```diff
@@ -97,8 +98,7 @@ namespace collision_detection
       EigenSTL::vector_Affine3d          shape_poses_;
     };
 
-    typedef boost::shared_ptr<Object> ObjectPtr;
-    typedef boost::shared_ptr<Object> ObjectConstPtr;
```

Comment:
`ObjectConstPtr` was not actually a `ConstPtr`. It is now, may break files who need mistakenly use  `ObjectConstPtr` as `ObjectPtr`.